components/script/dom/htmlscriptelement.rs

```diff
@@ -274,7 +274,8 @@ impl HTMLScriptElement {
 
         let window = window_from_node(self);
         let window = window.r();
```

Comment:
`window` is now only used for the `networking_task_source` call below. Please remove these two variables and use `document.window()` below.